ccd-definition/State/State-nonprod.json

```diff
@@ -8,5 +8,15 @@
     "DisplayOrder": 4,
     "TitleDisplay": "# ${caseName}\n## **FamilyMan ID: ${familyManCaseNumber}**\n## **CCD ID: #${[CASE_REFERENCE]}**",
     "Comment": "PREPARE_FOR_HEARING state name changed as part of FPLA-1920. CaseStateID naming remains to avoid break existing cases."
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "ISSUE_RESOLUTION",
+    "Name": "Issue Resolution",
+    "Description": "Issue resolution case state - Triggered via CMO event and hearing is of issue resolution type",
+    "DisplayOrder": 8,
+    "TitleDisplay": "# ${caseName}\n## **FamilyMan ID: ${familyManCaseNumber}**\n## **CCD ID: #${[CASE_REFERENCE]}**",
```

Comment:
Did we decide this was the correct display order for issue resolution? I would imagine it would come after prepare for hearing state and before closed, deleted and returned. Correct me if wrong though. :-)

Reply:
Good point, i'll raise with Eilis.